.github/workflows/w3c-integration-test.yml

```diff
@@ -0,0 +1,18 @@
+name: Run w3c tests on push
```

Comment:
Just curious to know, let's say in the future if we decide to extend this workflow to other propagators (such as `B3Propagator`), how would we go about it? Do we need to add new workflows or update this one?

Reply:
We could do either. Adding the support for that in the server should be easy if we have it accept a command-line argument, which would determine which propagator to use. Then, we can either add a new step in the workflow that tests b3, or add a new workflow.

Reply:
I was thinking it'd just be a new step.